crates/holochain/src/conductor/metrics.rs

```diff
@@ -0,0 +1,30 @@
+use opentelemetry_api::{global::meter_with_version, metrics::*};
+
+pub type P2pEventDurationMetric = Histogram<f64>;
+pub type PostCommitDurationMetric = Histogram<f64>;
+
+pub fn create_p2p_event_duration_metric() -> P2pEventDurationMetric {
```

Comment:
mmh, a trait for this perhaps?

Reply:
A trait? I'm not quite sure what you mean here?

Reply:
I mean a trait for the multiple metric types, so that future metrics will be consistent? They're all type aliases I see now, so perhaps it doesn't make sense to specify a trait. Decide as you see fit =)